CHANGELOG/CHANGELOG-2.2.md

```diff
@@ -0,0 +1,68 @@
+# Release notes for v2.2.0
+
+[Documentation](https://kubernetes-csi.github.io)
+# Changelog since v2.1.0
+
+## Changes by Kind
+
+### Feature
+
+- Added --logging-format to set log formatter (default: text) ([#129](https://github.com/kubernetes-csi/node-driver-registrar/pull/129), [@aramase](https://github.com/aramase))
```

Comment:
That PR hasn't been merged. Why does it show up here?

Reply:
Good catch! It comes from [release-tools import](https://github.com/kubernetes-csi/node-driver-registrar/commit/9e79ae460fa4691fc3530f08d54ab5cbf957e7ca) that has `Merge pull request #129 from pohly/squash-documentation`.

I removed the line.

Reply:
Let's discuss in https://kubernetes.slack.com/archives/C8EJ01Z46/p1619522100167200